format/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+[ -z "$TR_SKIP_HOOKS" ] || exit 0
+
+root="$(git rev-parse --show-toplevel)"
+"${root}/format/format.sh" --staged --test || exit 1
```

Comment:
Why do we mask the original exit code? It's non-zero already, no point in checking for it I guess...
```suggestion
exec "${root}/format/format.sh" --staged --test
```

Reply:
I think the `|| exit 1` is an unnecessary leftover from an earlier draft.  Good catch.